migration-helper/modelmesh-to-raw.sh

```
@@ -0,0 +1,2025 @@
+}
+
+# List InferenceServices and get user selection
+list_and_select_inference_services() {
+    echo "🔍 Discovering InferenceServices in source namespace '$FROM_NS'..."
+
+    # Initialize variables to avoid unset variable errors
+    local index=0
+    local isvc_count=0
+
+    # Get all InferenceServices in the source namespace
+    if ! local isvc_list=$(oc get inferenceservice -n "$FROM_NS" -o yaml 2>/dev/null); then
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)